### PR TITLE
Setup bundlemon

### PR DESCRIPTION
**Pull request by armanbilge**
Trying out a new plugin, thoughts?
https://github.com/armanbilge/sbt-bundlemon/

### CODE REVIEW

**Comment by cquiroz** [comment]
Interesting, What is building to compare the size? tests? demo?

**Comment by cquiroz** [comment]
Sounds good, Maybe the AGSSampleApp is more interesting though

**Comment by cquiroz** [comment]
Do we have a way to tell (in text or gaphically) what contributes to the size?

**Comment by armanbilge** [comment]
Is `AGSSampleApp` in this project? I was thinking we can enable it on many projects now, since the plugin makes it easy.

> Do we have a way to tell (in text or gaphically) what contributes to the size?

No. I guess we'd have to integrate with source-map-explorer for that. But if we start tracking this, at least we'll know when dependencies are increasing our artifact size.

**Comment by armanbilge** [comment]
Sure. You can enable the plugin on multiple sub-modules and it will track them separately. See http4s-dom where it tracks each Scala version independently.
- https://github.com/http4s/http4s-dom/pull/176

**Review by cquiroz — APPROVED** [review]
Seems like a great idea. LGTM

File: build.sbt [hunk 1]
```diff
@@ -96,15 +96,24 @@ lazy val tests = crossProject(JVMPlatform, JSPlatform)
     jsEnv                           := {
       import org.scalajs.jsenv.nodejs.NodeJSEnv
       new NodeJSEnv(NodeJSEnv.Config().withArgs(List("--experimental-fetch")))
-    }
+    },
+    bundleMonCheckRun               := true
```
[the remark this answers is not in the document]
Comment (armanbilge) [reply build.sbt:100]: It's measuring gzipped `fullLinkJS`

https://github.com/armanbilge/sbt-bundlemon/blob/dcbe99179e2ef806787703e4f1c4c30e5571d011/bundlemon/src/main/scala/com/armanbilge/sbt/BundleMonPlugin.scala#L65

https://github.com/armanbilge/sbt-bundlemon/blob/dcbe99179e2ef806787703e4f1c4c30e5571d011/bundlemon/src/main/scala/com/armanbilge/sbt/BundleMonPlugin.scala#L70-L74

File: build.sbt [hunk 2]
```diff
@@ -98,6 +98,7 @@ lazy val tests = crossProject(JVMPlatform, JSPlatform)
       new NodeJSEnv(NodeJSEnv.Config().withArgs(List("--experimental-fetch")))
     }
   )
+  .jsConfigure(_.enablePlugins(BundleMonPlugin))
```
Comment (armanbilge) [inline build.sbt:101]: 1-line setup 😃

File: build.sbt (hunk 1 above)
[the remark this answers is not in the document]
Comment (rpiaggio) [reply build.sbt]: > It's measuring gzipped fullLinkJS

Can we use brotli instead?

**Pull request merged**